app/main.py

```diff
@@ -6,10 +6,18 @@
 from app.sdxftp import SDXFTP
 from sdc.rabbit.publisher import QueuePublisher
 from sdc.rabbit.exceptions import QuarantinableError, RetryableError
+from ftplib import Error
 
 from sdc.rabbit.consumers import MessageConsumer
 
+import tornado.web
+from tornado.httpclient import HTTPClient
+import threading
+import json
+import os
+
 logger = create_and_wrap_logger(__name__)
+healthcheck_delay = 300
```

Comment:
Maybe make name more explicit too:
```
HEALTHCHECK_DELAY_SECONDS = 300
```

Can otherwise get confusing when you move between languages etc that have different default resolutions